app/models/post.js

```diff
@@ -0,0 +1,92 @@
+import DS from 'ember-data';
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.